lib/User/Backend.php

```diff
@@ -149,6 +149,52 @@ public function getLogoutUrl(): string {
 		);
 	}
 
+	/**
+	 * Return user data from the idp
+	 * Inspired by user_saml
+	 */
+	public function getUserData(): array {
```

Comment:
I don't see this being called, or is it needed from somewhere externally?

Reply:
Ah found in https://github.com/nextcloud/globalsiteselector/blob/331f24c43b565ca9a1d4b973a37248b3b563103d/lib/Master.php#L129 for SAML

Could have a cleaner API with something like a IGSUserBackend interface and typed methods to get the needed user data, but that is probably out of scope ;)